.github/workflows/ci.yml

```diff
@@ -93,24 +93,26 @@ jobs:
           fi
           sudo apt update && sudo apt install -y "${PKGS[@]}"
 
-      - name: Install gcc-11
+      - name: Install gcc-12
         if:  matrix.compiler == 'gcc'
         run: |
-          sudo apt install -y gcc-11
-          sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-11 100
-          sudo update-alternatives --set gcc /usr/bin/gcc-11
+          GCC_VER=12
```

Comment:
I think should set `GCC_VER` (and `CLANG_VER`) at `env:` (L22~) section; since it enable to describe `name` of the step as `Install gcc-${{ env.GCC_VER }}`.

Reply:
Done!